tests/cupy_tests/math_tests/test_misc.py

```diff
@@ -255,6 +255,15 @@ def test_nan_to_num_broadcast(self, kwarg):
             with pytest.raises(ValueError):
                 xp.nan_to_num(0.0, **{kwarg: y})
 
+    def test_real_if_close(self):
+        x = cupy.random.rand(10) + 1e-15j
+        assert cupy.all(cupy.real_if_close(x) == numpy.real_if_close(x))
```

Comment:
`numpy.real_if_close` calls `cupy.real_if_close` internally if `cupy.ndarray` is given as an input, so this test does not compare NumPy's and CuPy's results correctly.

Reply:
How about using `testing.numpy_cupy_array_equal`?

Reply:
> `numpy.real_if_close` calls `cupy.real_if_close` internally if `cupy.ndarray` is given as an input,

Ouch! Gotcha. 

> How about using `testing.numpy_cupy_array_equal`?

Alrighty. Will look a little deeper into the testing module as well to avoid mistakes in the future.